public/apps/account/tenant-switch-panel.tsx

```diff
@@ -169,20 +181,7 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
     },
     {
       id: CUSTOM_TENANT_RADIO_ID,
-      label: (
-        <>
-          Choose from custom
-          <EuiSuperSelect
```

Comment:
Looks like previously super select is part of the label. Now it is outside of the radio group. Probably we can try to move the single select here and try again.

Reply:
It is originally here and I remember it was the same. The reason I move out is: the radio button eats up the event with combo box so clicking in the combo box will select radio instead of show the drop down list.

Reply:
If radio button will intercept the event, a potential workaround may be trying to set value of combo box in onTenantSwitchRadioChange

Reply:
We don't know what value to set, we need to show the drop down list and let the user to select an option.